Makefile

```diff
@@ -182,3 +182,15 @@ appimage-%:
 	bash scripts/genappimage.sh $*
 
 .PHONY: test clean distclean nvim libnvim cmake deps install appimage checkprefix benchmark $(FORMAT) $(LINT) $(TEST)
+
+.PHONY: emmylua-check
+emmylua-check: luv
+	-emmylua_check \
+		--output-format=json \
+		--config=$(PWD)/.luarc.json \
```

Comment:
You probably want to use `CURDIR` instead of `PWD`. The latter will be incorrect with `make -C`. In fact, you basically never want to use `PWD`.

Reply:
Thanks. I'll make sure to use `CURDIR` instead from now on. This target is only temporary. The final form will be in cmake.